### PR TITLE
Add scene_id to the dataset methods parameters list

### PR DESCRIPTION
**Pull request by st235**

### CODE REVIEW

File: patch_scene.py [hunk 1]
```diff
@@ -45,7 +45,8 @@ def __patch_scene(scene_id: str,
 
         assert instance not in instance_accumulated_clouds_lookup
 
-        accumulated_point_cloud = point_cloud_accumulator.merge(instance_id=instance,
+        accumulated_point_cloud = point_cloud_accumulator.merge(scene_id=scene_id,
```
Comment (mariarzv) [inline patch_scene.py:48]: maybe frame_id could be here too? it would be useful for Gedi

File: src/accumulation/point_cloud_accumulator.py [hunk 2]
```diff
@@ -20,10 +20,13 @@ def __init__(self,
         self.__dataset = dataset
 
     def merge(self,
+              scene_id: str,
```
Comment (mariarzv) [inline src/accumulation/point_cloud_accumulator.py:23]: same should apply for gedi accumulator then, I could do it?

File: patch_scene.py (hunk 1 above)
Comment (st235) [reply patch_scene.py:48]: Point cloud accumulator merges instance across the entire scene, i.e. across all frames. You have `frame_id` availale inside.

File: src/accumulation/point_cloud_accumulator.py (hunk 2 above)
Comment (st235) [reply src/accumulation/point_cloud_accumulator.py:23]: We don't have `gedi accumulator`, right? We have only `gedi accumulation strategy`. Strategies don't work with frames they work with point clouds: a strategy merges merges `old` point cloud (i.e. accumulated) with a point cloud from the current frame. What do you think?

Comment (mariarzv) [reply src/accumulation/point_cloud_accumulator.py:23]: For Gedi, we need to know which object pointcloud to register to, if we register them with chain logic, similar to geometric accumulation, we run the risk of accumulating registration error as well. Therefore, current registration workflow registers all the instance objects to a chosen "first".  To be able to differentiate between point clouds we need some info like frame_id

Comment (st235) [reply src/accumulation/point_cloud_accumulator.py:23]: I believe strategies should not make a distinction between frames. If you need to start from different frame or use a different step we can modify `a constructor` + `logic` of the `point_cloud_accumulator` to change the behaviour as we want. We will still have "chain" merge logic even if we are going to merge the clouds across different frames, right?
I am not sure I get what you want to achieve. Can you, please, explain it to me again and we can try to work from there?

Comment (mariarzv) [reply src/accumulation/point_cloud_accumulator.py:23]: explained offline. ideally we need the info of the "first" instance appearence

Comment (st235) [reply src/accumulation/point_cloud_accumulator.py:23]: agreed to pass the initial cloud as a parameter of `point cloud strategy`

**Review by mariarzv — APPROVED** [review]
The concerns mentioned require different changes and will be addressed in another PR

**Pull request merged**